Sources/System/FileControlRaw.swift

```diff
@@ -0,0 +1,726 @@
+    ///
+    /// The corresponding C constant is `F_SETLK`.
+    @_alwaysEmitIntoClient
+    public static var setLock: Command { Command(F_SETLK) }
+
+    /// Wait if blocked.
+    ///
+    /// Note: A Swiftier wrapper is `FileDescriptor.setLock(_:_:)`.
+    ///
+    /// The corresponding C constant is `F_SETLKW`.
+    @_alwaysEmitIntoClient
+    public static var setLockWait: Command { Command(F_SETLKW) }
+
+    /// Wait if blocked, return on timeout.
+    ///
+    /// TODO: A Swiftier wrapper
+    ///
+    /// The corresponding C constant is `F_SETLKWTIMEOUT`.
+    @_alwaysEmitIntoClient
+    public static var setLockWaitTimout: Command {
+      Command(F_SETLKWTIMEOUT)
+    }
+
+    /// ??? TODO: Where is this documented?
+    ///
+    /// The corresponding C constant is `F_FLUSH_DATA`.
+    @_alwaysEmitIntoClient
+    public static var flushData: Command {
+      Command(F_FLUSH_DATA)
+    }
+
+    /// Used for regression test.
+    ///
+    /// The corresponding C constant is `F_CHKCLEAN`.
+    @_alwaysEmitIntoClient
+    public static var checkClean: Command { Command(F_CHKCLEAN) }
+
+    /// Preallocate storage.
+    ///
+    /// The corresponding C constant is `F_PREALLOCATE`.
+    @_alwaysEmitIntoClient
+    public static var preallocate: Command {
+      Command(F_PREALLOCATE)
+    }
+
+    /// Truncate a file. Equivalent to calling truncate(2).
+    ///
+    /// The corresponding C constant is `F_SETSIZE`.
+    @_alwaysEmitIntoClient
+    public static var setSize: Command { Command(F_SETSIZE) }
+
+    /// Issue an advisory read async with no copy to user.
+    ///
+    /// The corresponding C constant is `F_RDADVISE`.
+    @_alwaysEmitIntoClient
+    public static var readAdvise: Command { Command(F_RDADVISE) }
+
+    /// Turn read ahead off/on for this fd.
+    ///
+    /// The corresponding C constant is `F_RDAHEAD`.
+    @_alwaysEmitIntoClient
+    public static var readAhead: Command { Command(F_RDAHEAD) }
+
+    ///
+    /// Header says: 46,47 used to be F_READBOOTSTRAP and F_WRITEBOOTSTRAP
+    /// FIXME: What do we do here?
+    ///
+
+    /// Turn data caching off/on for this fd.
+    ///
+    /// The corresponding C constant is `F_NOCACHE`.
+    @_alwaysEmitIntoClient
+    public static var noCache: Command { Command(F_NOCACHE) }
+
+    /// File offset to device offset.
+    ///
+    /// The corresponding C constant is `F_LOG2PHYS`.
+    @_alwaysEmitIntoClient
+    public static var log2phys: Command { Command(F_LOG2PHYS) }
+
+    /// Return the full path of the fd.
+    ///
+    /// Note: A Swiftier wrapper is `FileDescriptor.getPath(_:)`.
+    ///
+    /// The corresponding C constant is `F_GETPATH`.
+    @_alwaysEmitIntoClient
+    public static var getPath: Command { Command(F_GETPATH) }
+
+    /// Fsync + ask the drive to flush to the media.
+    ///
+    /// The corresponding C constant is `F_FULLFSYNC`.
+    @_alwaysEmitIntoClient
+    public static var fullFsync: Command { Command(F_FULLFSYNC) }
```

Comment:
What are we going to call `fsync(2)`? I think I'd suggest `FileDescriptor.sync` or `synchronize`. We should use the same name here.